StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElementContainerView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  EmbeddedPaymentElementContainerView.swift
+//  StripePaymentSheet
+//
+//  Created by Yuki Tokuhiro on 10/16/24.
+//
+import UIKit
+
+/// The view that's vended to the merchant, containing the embedded view.  We use this to be able to swap out the embedded view with an animation when `update` is called.
+class EmbeddedPaymentElementContainerView: UIView {
+    var updateSuperviewHeight: () -> Void = {}
+    var view: UIView
```

Comment:
Any reason these aren't private (excluding `updateSuperviewHeight`, although that feels like it should come in at init time and be optional)? Also wondering it it ever makes sense to use this class when `view` is not of type `EmbeddedPaymentMethodsView`. Should it just be `EmbeddedPaymentMethodsView` over `UIView` in this class?

Reply:
`updateSuperviewHeight` is a var for convenience; `self` isn't available when `EmbeddedPaymentElement` initializes this.

Will make them private & change the UIView type though!